lib/checkexceptionsafety.cpp

```diff
@@ -393,3 +394,39 @@ void CheckExceptionSafety::rethrowNoCurrentExceptionError(const Token *tok)
                 " More: https://isocpp.org/wiki/faq/exceptions#throw-without-an-object",
                 CWE480, Certainty::normal);
 }
+
+void CheckExceptionSafety::throwIsMissing()
+{
+    static const std::unordered_set<std::string> stdExceptions{"exception",
```

Comment:
For future reference only.

We should no longer maintain such lists in the source (some exceptions do apply) but they should be part of a library configuration. This would also allow us to easily extend this for libraries beside the `std` one.